app/src/test/java/me/saket/dank/utils/UrlParserTest.java

```diff
@@ -480,6 +480,17 @@ public void parseStreamableUrl() {
     assertThat(((StreamableUnresolvedLink) parsedLink).videoId()).isEqualTo("jawcl");
   }
 
+  @Test
+  public void parseYouTubeUrl() {
```

Comment:
Could also add "negative tests" where you test that similar looking links don't parse as YouTube urls.